eng/pipelines/templates/steps/build-test.yml

```diff
@@ -26,27 +26,27 @@ steps:
   - template: /eng/common/pipelines/templates/steps/verify-agent-os.yml
 
   - script: |
-      python -m pip install pip == 20.1
-      pip install -r eng/ci_tools.txt
-      pip --version
+      python -m pip install -r eng/ci_tools.txt
```

Comment:
I need to re-add `pip` here. But the default currently is `20.1`. I will do it in a follow-up PR.